src/components/bulkConcepts/bulkConceptList.jsx

```diff
@@ -81,8 +38,5 @@ BulkConceptList.propTypes = {
   cielConcepts: PropTypes.arrayOf(PropTypes.shape({
     name: PropTypes.string,
   })).isRequired,
-  handleSelect: PropTypes.func.isRequired,
-  conceptLimit: PropTypes.number.isRequired,
```

Comment:
Is this related to the ticket?

Reply:
Yes, It was part of the table I removed. So it was no longer necessary.